docs/developer-docs/functionality/sns/index.md

```diff
@@ -0,0 +1,74 @@
+# How to launch an SNS
```

Comment:
Should this page start with what an SNS is?

Reply:
There will be another page explaining what an SNS is /functionality/sns and this here should only be the introduction to how to deploy an SNS.